apps/dashboard/app/(app)/authorization/permissions/[permissionId]/page.tsx

```diff
@@ -85,12 +85,12 @@ export default async function RolesPage(props: Props) {
           >
             <Tooltip>
               <TooltipTrigger asChild>
-                <div className="flex items-center justify-between gap-2 truncate">
-                  <span className="truncate">{permission.name}</span>
-                  <div>
-                    <CopyButton value={permission.name} />
-                  </div>
-                </div>
+                 <div className="flex items-center justify-between gap-2 w-full truncate">
+        <span className="truncate max-w-[calc(100%-32px)]">{permission.name}</span>
```

Comment:
This is too brittle, if we now change the size of the copy button, this will break

a simple w-full might work